exchange/adapter_util.go

```diff
@@ -33,6 +33,10 @@ func buildBidders(infos config.BidderInfos, builders map[openrtb_ext.BidderName]
 	var errs []error
 
 	for bidder, info := range infos {
+		if len(info.AliasOf) > 0 {
+			errs = append(errs, fmt.Errorf("This feature is currently under development"))
```

Comment:
For my own understanding since I'm getting confused, why are we only introducing this feature if it's not able to be used yet?

Reply:
This check will make sure that no one should be using this feature yet as it is currently under development. So, if someone sets `AliasOf` field on BidderInfo struct, this check will protect it from happening. 
To further develop this feature, we need an `AliasOf` field on BidderInfo struct. For example - adding validation checks like `parent bidder of an alias cannot be an alias` or `capabilities validation` (capabilities of an alias should be subset of parent` etc. 
In this way, we are slowly building this feature and making sure that feature is not being used by anyone while we are building it.